packages/backend/discovery/vector/ethereum/config.jsonc

```diff
@@ -7,26 +7,12 @@
     "0x02993cdC11213985b9B13224f3aF289F03bf298d": "Vector",
     "0x3B6041173B80E77f038f3F2C0f9744f04837185e": "SuccinctGatewaySP1",
     "0xCafEf00d348Adbd57c37d1B77e0619C6244C6878": "SuccinctGatewaySP1Multisig",
-    "0x7F2f87B0Efc66Fea0b7c30C61654E53C37993666": "AvailMultisig"
+    "0x7F2f87B0Efc66Fea0b7c30C61654E53C37993666": "AvailMultisig",
+    "0x44eb418a966ff47f5af6f48aea6afde0bf193a8d": "Relayer"
   },
   "overrides": {
     "Vector": {
-      "extends": "succinct/SP1Vector",
-      "fields": {
-        "relayers": {
-          "handler": {
-            "type": "hardcoded",
-            "value": ["0x44eb418a966ff47f5af6f48aea6afde0bf193a8d"]
-          }
-        },
-        "isRelayerApproved": {
-          "handler": {
-            "type": "call",
-            "method": "function approvedRelayers(address) view returns (bool)",
-            "args": ["0x44eb418a966ff47f5af6f48aea6afde0bf193a8d"]
-          }
-        }
-      }
+      "extends": "succinct/SP1Vector"
```

Comment:
When working with config.jsonc files, please remove all "extends" keywords (and also don't add or approve new "extends"). It's an old approach that we're getting rid of. Instead, add flattened .sol file to template shape, so that it's discovered by implementation match.

Reply:
https://github.com/l2beat/l2beat/pull/6052/commits/e606330a4a3b6025053a4d960236b9fd26dc5047
https://github.com/l2beat/l2beat/pull/6052/commits/16ff8bc3b220b425db62556d9a5e71cb2d62b464